tests/test_bokky_poo_bahs_date-time-library.py

```diff
@@ -31,6 +33,23 @@ def test_timestamp_to_date_automated(date_time_library, start_date, end_date):
         current_date += relativedelta(days=+1)
 
 
+@given(
+    timestamp=strategy(
+        "uint256",
+        min_value=get_timestamp_from_date(2020, 1, 1),
+        max_value=get_timestamp_from_date(2050, 1, 1),
```

Comment:
Again, regarding the aim for dense coverage, 2050 sounds quite excess for me. Negligible probability this code would run till 2050.